src/WebServerManager.cpp

```
@@ -1972,39 +1934,17 @@ int WebServerManager::enrichFindOrCreateFilament(WiFiClient& client, HTTPClient&
-                        // RGB-only compare — Spoolman color_hex may carry an alpha suffix
-                        if (strlen(fc) < 6 || strncasecmp(fc, colorCmp, 6) != 0) continue;
-                    }
-                    filamentId = f["id"] | -1;
-                    break;
-                }
-            }
-        }
-        http.end();
-
+        const char* colorCmp = colorHex;
+        if (colorCmp[0] == '#') colorCmp++;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)